packages/expo-cli/src/commands/build/BaseBuilder.ts

```diff
@@ -78,7 +57,7 @@ export default class BaseBuilder {
   }
 
   async prepareProjectInfo(): Promise<void> {
-    // always use local json to unify behaviour between regular apps and self hosted ones
+    // always use local json to unify behavior between regular apps and self hosted ones
```

Comment:
it's behaviour everywhere except USA!

Reply:
I have a spellcheck linter installed in code. [also](https://i.imgflip.com/3ggvsj.png)